DOCUMENTATION.md

```diff
@@ -2933,72 +2933,7 @@ Please visit the [Giraffe.ViewEngine](https://github.com/giraffe-fsharp/Giraffe.
-
-To use `Utf8Json.Serializer` instead of `NewtonsoftJson.Serializer`, register a new dependency of type `Json.ISerializer` during application configuration:
-
-```fsharp
-let configureServices (services : IServiceCollection) =
-    // First register all default Giraffe dependencies
-    services.AddGiraffe() |> ignore
-
-    // Now register Utf8Json.Serializer
-    this.AddSingleton<Json.ISerializer>(Utf8Json.Serializer(Utf8Json.Serializer.DefaultResolver)) |> ignore
-```
-
-Or to use `SystemTextJson.Serializer` instead of `NewtonsoftJson.Serializer`, register a new dependency of type `Json.ISerializer` during application configuration:
-
-```fsharp
-let configureServices (services : IServiceCollection) =
-    // First register all default Giraffe dependencies
-    services.AddGiraffe() |> ignore
-
-    let serializationOptions = SystemTextJson.Serializer.DefaultOptions
-    // Optionally use `FSharp.SystemTextJson` (requires `FSharp.SystemTextJson` package reference)
-    serializationOptions.Converters.Add(JsonFSharpConverter(JsonUnionEncoding.FSharpLuLike))
-    // Now register SystemTextJson.Serializer
-    services.AddSingleton<Json.ISerializer>(SystemTextJson.Serializer(serializationOptions)) |> ignore
-```
-
-
-#### Customizing JsonSerializerSettings
-
-You can change the default `JsonSerializerSettings` of the `NewtonsoftJson.Serializer` by registering a new instance of `NewtonsoftJson.Serializer` during application startup. For example, the [`Microsoft.FSharpLu` project](https://github.com/Microsoft/fsharplu/wiki/fsharplu.json) provides a Json.NET converter (`CompactUnionJsonConverter`) that serializes and deserializes `Option`s and discriminated unions much more succinctly. If you wanted to use it, and set the culture to German, your configuration would look something like:
-
-```fsharp
-let configureServices (services : IServiceCollection) =
-    // First register all default Giraffe dependencies
-    services.AddGiraffe() |> ignore
-
-    // Now customize only the Json.ISerializer by providing a custom
-    // object of JsonSerializerSettings
-    let customSettings = JsonSerializerSettings(
-        Culture = CultureInfo("de-DE"))
-    customSettings.Converters.Add(CompactUnionJsonConverter(true))
-
-    services.AddSingleton<Json.ISerializer>(
-        NewtonsoftJson.Serializer(customSettings)) |> ignore
-
-[<EntryPoint>]
-let main _ =
-    WebHost.CreateDefaultBuilder()
-        .Configure(Action<IApplicationBuilder> configureApp)
-        .ConfigureServices(configureServices)
-        .ConfigureLogging(configureLogging)
-        .Build()
-        .Run()
-    0
-```
+By default Giraffe uses `System.Text.Json` for (de-)serializing JSON content. An application can modify the default serializer by registering a new dependency which implements the `Json.ISerializer` interface during application startup.
```

Comment:
TODO: add documentation explaining how someone could configure `Newtonsoft.Json` in case it's still necessary.

Reply:
@64J0 I've added it now.

Reply:
Do you mind rebasing this PR too?

Reply:
Sure thing